src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -8,11 +8,13 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dDriverAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.

Inheritance is not used (no super drawLine calls). Should be removed!